exercises/grade-school/grade_school_test.rb

```diff
@@ -4,85 +4,104 @@
 require_relative 'grade_school'
 
 class SchoolTest < Minitest::Test
-  attr_reader :school
-
-  def setup
-    @school = School.new
-  end
-
-  def test_an_empty_school
-    assert_equal({}, school.to_h)
+  def test_empty_grade
+    subject = School.new
```

Comment:
Discussion of the appropriate return values should probably move to #299 rather than continuing here.